buildscripts/build.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
```

Comment:
It will be good to move away from these build scripts. Can we have makefile & dockerfile to ensure builds, images, tests, integration tests, etc

Reply:
will see that in next PR.